### PR TITLE
ProductCards fetches from SQL db

### PR DESCRIPTION
**Pull request by WillTrem**
ProductCards now fetches directly products from the sql database.

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ app.use(express.json());
 
 app.use(function(req, res, next) {
 	res.header("Access-Control-Allow-Origin", "https://heagle.herokuapp.com"); // update to match the domain you will make the request from
-	//res.header("Access-Control-Allow-Origin", "http://localhost:3000"); // update to match the domain you will make the request from
+	res.header("Access-Control-Allow-Origin", "http://localhost:3000"); // update to match the domain you will make the request from
```
Comment (Its-Alann) [inline server/server.js:13]: Should both of these be uncommented at the same time?

**Review by Its-Alann — APPROVED** [review]
Yeah, everything seems fine with me. I just have one question about one of the files. You should be able to see the question. Thanks

File: server/server.js (hunk 1 above)
Comment (WillTrem) [reply server/server.js:13]: Yeah from my testing both can be uncommented at the same time, basically I think it is like a whitelist for certain urls

Comment (Its-Alann) [reply server/server.js:13]: Ok sounds good, thanks

**Review by Its-Alann — APPROVED** [review]
Seems good

**Review by dange2 — APPROVED** [review]
Looks great, thanks Will!

**Pull request merged**